### PR TITLE
Adds new stage command

### PR DESCRIPTION
**Pull request by cossou**

### CODE REVIEW

File: commands/stage.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  const cwd = path.resolve('./');
+  const service = path.basename(cwd);
+  const environment = 'staging';
+  const herokuAppName = `test-${service}`;
+  const herokuRemote = `${environment}\thttps://git.heroku.com/${herokuAppName}.git`;
+  let branchName;
+
+  // Check if we are on a Git directory and get current branch
+  try {
+    const { stdout } = await execa('git', ['symbolic-ref', '--short', '-q', 'HEAD'], {
```
Comment (ebdrup) [inline commands/stage.js:29]: `const { stdout : branchName } = await execa...` ?

File: commands/stage.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    console.error(` Looks like "${cwd}" is not a git directory.`);
+    console.log(e.shortMessage);
+    process.exit(e.exitCode);
+  }
+
+  const branch = selectedBranch || branchName;
+
+  console.log();
+  console.log(` Current working dir: `.reset + `${cwd}`.bgGreen.black);
+  console.log(` Deploying branch:    `.reset + `${branch}`.bgGreen.black);
```
Comment (ebdrup) [inline commands/stage.js:43]: If I'm not mistaken there is a print function for this kind of output

File: commands/stage.js (hunk 1 above)
Comment (cossou) [reply commands/stage.js:29]: Can't do that. `branchName` is defined before this call.

File: commands/stage.js (hunk 2 above)
Comment (cossou) [reply commands/stage.js:43]: I couldn't find the print function. Maybe you are thinking about CLI :D

**Review by samfcmc — APPROVED** [review]
🚀

**Pull request merged**